### PR TITLE
poc: can we use tailwind

### PR DESCRIPTION
**Pull request by danielgent**
#### Done in this PR
Tailwind proof-of-concept!

Kind of works :smile: 

image

#### Next
1. I'll check this form actually works for changing the password
2. I could do some crude error outputting
3. Get Jonatan involved to start styling?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,6 +67,7 @@
     "zod": "^3.22.4"
   },
   "devDependencies": {
+    "@ape-egg/tailwind-rows-columns": "^1.0.2",
```
Comment (danielgent) [inline package.json:70]: Kim's library that he loves :see_no_evil:

File: scripts/upload-css.sh
Comment (danielgent) [inline scripts/upload-css.sh:1]: works!  I'm not sure what other solution we have

It's annoying to have to do this and we might forget but it's not the end of the world

File: src/app.ts [hunk 2]
```diff
@@ -80,6 +80,25 @@ app.get(
...
+    );
+
+    if (!response) {
+      throw new Error("Template not found");
+    }
+
+    const templateString = await response.text();
+
+    return ctx.text(templateString, 200, {
+      "content-type": "text/css",
```
Comment (danielgent) [inline src/app.ts:97]: :see_no_evil:

Comment (markusahlstrand) [reply src/app.ts:97]: Maybe we should add some cache headers in here but we can do that later :)

**Review by markusahlstrand — APPROVED** [review]
LGTM!

**Pull request merged**